.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,110 @@
+---
+name: E2E Tests
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+    paths-ignore:
+      - .circlecli/**
+      - meta/**
+      - .gitignore
+      - .markdownlint.yml
+      - .yamllint
+      - ansible.cfg.galaxy
+      - CHANGELOG.md
+      - LICENSE
+      - poetry.lock
+      - pyproject.toml
+      - README.md
+      - Vagrantfile
+  push:
+    branches:
+      - master
+      - ansible-molecule-4
+  schedule:
+    - cron: "0 1 * * *"
+  workflow_dispatch:
+
+env:
+  PY_COLORS: true
+  ANSIBLE_FORCE_COLOR: true
+  MOLECULE_DEBUG: false
+  MOLECULE_VERBOSITY: 0
+  MOLECULE_NO_LOG: true
+
+concurrency:
+  group: e2e
+  cancel-in-progress: false
+
+jobs:
+  molecule:
+    runs-on: ubuntu-20.04
+    strategy:
+      max-parallel: 2
+      fail-fast: false
+      matrix:
+        st2_repo:
+          - stable
+          - unstable
+        os:
+          - centos7
+          - ubuntu18
+          - rockylinux8
+          - ubuntu20
+        python:
+          # - version: 3.6
+          #   ansible-core: 2.11.12
+          #   molecule: 3.6.1
+          #   molecule-docker: 1.1.0
+          #   molecule-scenario: python36
+          - version: 3.8
+            ansible-core: 2.11.12
+            molecule: 3.6.1
+            molecule-docker: 1.1.0
+            molecule-scenario: python38
```

Comment:
```suggestion
        ansible:
          - 2.11.12
```

Please include only the ansible core version in the build matrix.

Everything else could be hidden and pinned somewhere.
In other words, we don't care about `molecule`, `molecule-docker`, or `molecule-scenario` in the build matrix.
We even don't care if the e2e tests are run with a test-kitchen or molecule.

It shouldn't be exposed in the CI status string here to reduce the information overhead:
![Screenshot 2023-01-02 at 19-10-07 CI Use Ansible Molecule for end-to-end testing (Rework) by mamercad · Pull Request #329 · StackStorm_ansible-st2](https://user-images.githubusercontent.com/1533818/210270759-6bae9d50-3f36-42b3-aa3b-e973c01c53ef.png)

Adding an ansible version in there would be already an improvement, - thanks for that work. 
![Screenshot 2023-01-02 at 19-12-06 Pull requests · StackStorm_ansible-st2](https://user-images.githubusercontent.com/1533818/210270899-17afdd0a-8bce-49c5-9650-03c8317222e7.png)
Let's just include only important info!

Reply:
Makes sense, I'll work on it.